src/api/mobility.ts

```diff
@@ -42,3 +53,48 @@ export const getVehicle = (
     })
     .then((res) => res.data.vehicles?.[0]);
 };
+
+export const getStations = (
+  {lat, lon, range, availableFormFactors, operators}: GetStationsQueryVariables,
+  opts?: AxiosRequestConfig,
+) => {
+  const url = '/bff/v2/mobility/stations';
+  const query = qs.stringify({
+    lat,
+    lon,
+    range: Math.ceil(range),
+    availableFormFactors,
+    operators,
+  });
+  return client
+    .get<GetStationsQuery>(stringifyUrl(url, query), {
+      ...opts,
```

Comment:
See the comment below ;)

Reply:
Yes, this is strictly not necessary, but really convenient when you need to test against a local BFF an have to add `baseUrl`:
```
{
    ...opts,
    baseUrl: 'http://localhost:8080'
}
```
I'll remove it, nevertheless. All minor things like this will eventually add up